scriptmodules/ports/gmloader.sh

```diff
@@ -13,19 +13,25 @@
 rp_module_id="gmloader"
 rp_module_desc="GMLoader - play GameMaker Studio games for Android on non-Android operating systems"
 rp_module_help="ROM Extensions: .apk .APK\n\nIncludes free games Maldita Castilla and Spelunky Classic HD. Use launch scripts as template for additional games."
-rp_module_repo="git https://github.com/JohnnyonFlame/droidports.git master faf3970"
+rp_module_repo="git https://github.com/JohnnyonFlame/droidports.git"
 rp_module_licence="GPL3 https://raw.githubusercontent.com/JohnnyonFlame/droidports/master/LICENSE.md"
 rp_module_section="exp"
-rp_module_flags="!all rpi4"
+rp_module_flags="!all rpi4 rpi3"
 
 function depends_gmloader() {
-    getDepends libopenal-dev libfreetype6-dev zlib1g-dev libbz2-dev libpng-dev libzip-dev libsdl2-image-dev cmake
+    local deps=(libopenal-dev libfreetype6-dev zlib1g-dev libbz2-dev libpng-dev libzip-dev libsdl2-image-dev cmake)
+    # for apk repacking
+    local deps+=(apktool default-jdk)
+    getDepends "${deps[@]}"
```

Comment:
Non-essential, but all "native" RetroPie modules use full word "depends" for this variable.